cluster/gce/config-test.sh

```diff
@@ -145,6 +145,16 @@ ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-influxdb}"
 # TODO(piosz) remove this option once Metrics Server became a stable thing.
 ENABLE_METRICS_SERVER="${KUBE_ENABLE_METRICS_SERVER:-true}"
 
+# Optional: Metadata agent to setup as part of the cluster bring up:
+#   none        - No metadata agent
+#   stackdriver - Stackdriver metadata agent
+# Metadata agent is a daemon set that provides metadata of kubernetes objects
+# running on the same node for exporting metrics and logs.
+ENABLE_METADATA_AGENT="${KUBE_ENABLE_METADATA_AGENT:-none}"
```

Comment:
If this is none in test by default, shouldn't it be enabled somewhere for testing?

Reply:
It is enabled specifically for instrumentation tests: https://github.com/kubernetes/test-infra/blob/master/jobs/env/ci-kubernetes-e2e-gce-stackdriver.env